extensions/amp-cache-url/0.1/test/test-amp-cache-url.js

```diff
@@ -29,5 +29,19 @@ describes.fakeWin(
         'https://amp-dev.cdn.ampproject.org/c/s/amp.dev/stories'
       );
     });
+
+    it('should not throw with empty url', async () => {
+      const cacheUrlService = new AmpCacheUrlService();
+      expect(
+        async () => await cacheUrlService.createCacheUrl('')
+      ).to.not.throw();
+    });
+
+    it('should not throw with invalid url', async () => {
+      const cacheUrlService = new AmpCacheUrlService();
+      expect(
+        async () => await cacheUrlService.createCacheUrl('invalid url')
+      ).to.not.throw();
+    });
```

Comment:
I'm not 100% confident in the specifics (haven't used this syntax in a bit) but Mocha has matchers for promises

```suggestion
    it('should not throw with empty url', async () => {
      const cacheUrlService = new AmpCacheUrlService();
      expect(cacheUrlService.createCacheUrl('')).to.not.be.rejected;
    });

    it('should not throw with invalid url', async () => {
      const cacheUrlService = new AmpCacheUrlService();
      expect(cacheUrlService.createCacheUrl('invalid url')).to.not.be.rejected;
    });
```

Since we can't use `async`/`await` directly in AMP code, testing the promises feels closer to the actual usage than testing the async wrapping, even though they're equivalent